README.md

```diff
@@ -4,19 +4,71 @@
 Evaluation kit for HEAR 2021 NeurIPS competition, using tasks from
 [hear-preprocess](https://github.com/neuralaudio/hear-preprocess).
 
+# hear-eval-kit
+Downstream evaluation on each task involves two
+steps:
+* computing audio embeddings
+* learning a shallow fully-connected predictor
+The first step's speed depends upon a variety of factors.
+The second step step's speed is relatively similar between models.
+Models with larger embeddings scale sub-linearly in training time
+(because of GPU optimizations) and linearly in hop-size (for
+event-based prediction tasks). The main hyperparameters controlling
+downstream training time are the maximum number of epochs and number
+of grid points for grid search.
```

Comment:
Maybe move this info on speed to later on in the readme? I don't think this is important for the introduction -- it's own little section below like "Notes on speed" or something.